addons/base_import/models.py

```diff
@@ -416,6 +537,68 @@ def _convert_import_data(self, record, fields, options, context=None):
 
         return data, import_fields
 
+    def _remove_currency_symbol(self, cr, uid, value):
+        value = value.strip()
+        float_regex = re.compile(r'([0-9.,]+)')
+        split_value = filter(None, float_regex.split(value))
+        if len(split_value) > 2:
+            # This is probably not a float
+            return False
+        if len(split_value) == 1:
+            if float_regex.search(split_value[0]) is not None:
+                return split_value[0]
+            return False 
+        else:
+            # String has been split in 2, locate which index contains the float and which does not
+            currency_index = 0
+            if float_regex.search(split_value[0]) is not None:
+                currency_index = 1
+            # Check that currency exists
+            currency = self.pool.get('res.currency').search(cr, uid, [('symbol', '=', split_value[currency_index])])
+            if len(currency):
+                return split_value[currency_index+1%2]
+            # Otherwise it is not a float with a currency symbol
+            return False
+
+    def _parse_float_from_data(self, cr, uid, data, index, name, options, context=None):
+        remove_parenthesis = remove_sign = False
```

Comment:
Why? remove_sign isn't used anywhere and remove_parenthesis is unconditionally set before it's used